app/views/fields/belongs_to/_form.html.erb

```diff
@@ -16,7 +16,11 @@ that displays all possible records to associate with.
 [1]: http://www.rubydoc.info/gems/administrate/Administrate/Field/BelongsTo
 %>
 
-<%= f.label field.permitted_attribute %>
-<%= f.select(field.permitted_attribute) do %>
-  <%= options_for_select(field.associated_resource_options, field.selected_option) %>
-<% end %>
+<div class="field-unit__label">
```

Comment:
I’m split on that, but mostly just to offer the ability to put a hint alongside the label, or a required flag, or any other element.